spec/forms/sufia/forms/work_form_spec.rb

```diff
@@ -17,4 +17,34 @@
       expect(form.ordered_work_members.size).to eq(2)
     end
   end
+
+  describe ".build_permitted_params" do
+    before do
+      allow(described_class).to receive(:model_class).and_return(GenericWork)
+    end
+    subject { described_class.build_permitted_params }
+    context "without mediated deposit" do
+      it { is_expected.to include(permissions_attributes: [:type, :name, :access, :id, :_destroy]) }
+    end
+  end
+
+  describe ".model_attributes" do
+    before do
+      allow(described_class).to receive(:model_class).and_return(GenericWork)
+    end
+    subject { described_class.model_attributes(ActionController::Parameters.new(attributes)) }
+    context "and a user is granted edit access" do
+      let(:attributes) { { permissions_attributes: [{ type: 'person', name: 'justin', access: 'edit' }] } }
+
+      context "without mediated deposit" do
+        it { is_expected.to eq ActionController::Parameters.new(permissions_attributes: [ActionController::Parameters.new(type: 'person', name: 'justin', access: 'edit')]).permit! }
+      end
+      context "with mediated deposit" do
+        before do
+          allow(Flipflop).to receive(:enable_mediated_deposit?).and_return(true)
+        end
+        it { is_expected.to eq ActionController::Parameters.new(permissions_attributes: []).permit! }
```

Comment:
it's `permit!` is required because it sets a `permitted=true` flag within the `Parameters` object. This is part of the `eq` comparison.